apps/rush-lib/src/data/test/jsonFiles/rushWithLockVersion.json

```diff
@@ -0,0 +1,18 @@
+{
+  "npmVersion": "4.5.0",
+  "rushMinimumVersion": "2.5.0",
+  "nodeSupportedVersionRange": ">=6.9.0 <7.0.0",
+  "projectFolderMinDepth": 1,
+  "projectFolderMaxDepth": 99,
+
+  "versionPolicies": [
```

Comment:
>versionPolicies [](start = 3, length = 15)

The word "policies" usually refers to enforcing that people behave a certain way.  For example, they must configure their Git email a certain way, their project folders must not be too deep, they must not use certain packages without approval, etc.

What you are describing here are configurations that describe the approach for how a package gets published.  You are also indirectly establishing a group of packages that will get published together, and whose version numbers may be the same.

So, maybe this field should be called "publishingSets" or "publishingGroups"?  And then each project will say which set it belongs to.  If it doesn't belong to a set, then it doesn't get published.

(We could also call them "versioningSets" or "versioningGroups", but if I understand right, this setting would *never* be assigned to a project that does not get published.  If so, the scenario is really publishing, and versioning is just an implementation detail.)